django/core/files/storage.py

```diff
@@ -140,24 +149,102 @@ def url(self, name):
     def accessed_time(self, name):
         """
         Returns the last accessed time (as datetime object) of the file
-        specified by name.
+        specified by name. Deprecated: use get_accessed_time() instead.
         """
+        warnings.warn(
+            'Storage.accessed_time() is deprecated. '
+            'Use Storage.get_accessed_time() instead.',
+            RemovedInDjango20Warning,
+            stacklevel=2,
+        )
         raise NotImplementedError('subclasses of Storage must provide an accessed_time() method')
 
     def created_time(self, name):
         """
         Returns the creation time (as datetime object) of the file
-        specified by name.
+        specified by name. Deprecated: use get_created_time() instead.
         """
+        warnings.warn(
+            'Storage.created_time() is deprecated. '
+            'Use Storage.get_created_time() instead.',
+            RemovedInDjango20Warning,
+            stacklevel=2,
+        )
         raise NotImplementedError('subclasses of Storage must provide a created_time() method')
 
     def modified_time(self, name):
         """
         Returns the last modified time (as datetime object) of the file
-        specified by name.
+        specified by name. Deprecated: use get_modified_time() instead.
         """
+        warnings.warn(
+            'Storage.modified_time() is deprecated. '
+            'Use Storage.get_modified_time() instead.',
+            RemovedInDjango20Warning,
+            stacklevel=2,
+        )
         raise NotImplementedError('subclasses of Storage must provide a modified_time() method')
 
+    def get_accessed_time(self, name):
+        """
+        Returns the last accessed time (as a datetime object) of the file
+        specified by name. The datetime object will be timezone-aware if
+        USE_TZ=True.
+        """
+        warnings.warn(
```

Comment:
Yes, I do. These are shims for when a third party backend hasn't updated to the new API, so there'll be deprecation warnings for them. Any up to date storage backend will override these methods without deprecation warnings (as `FileSystemStorage` does).

This is according to the grand plan (https://groups.google.com/forum/#!searchin/django-developers/timezone$20storage/django-developers/e6qY_qtjUDE/sFENDCYtAgAJ), which I perhaps should have linked to sooner, apologies :-/